python/eos/tasks.py

```diff
@@ -455,15 +455,15 @@ def sample_nested(analysis_file:str, posterior:str, base_directory:str='./', bou
     """
     analysis = analysis_file.analysis(posterior)
     results = analysis.sample_nested(bound=bound, nlive=nlive, dlogz=dlogz, maxiter=maxiter)
-    samples = _np.array(list(map(analysis._x_to_par, results.samples)))
+    samples = results.samples
     weights = _np.exp(results.logwt - results.logz[-1])
     eos.data.DynestyResults.create(os.path.join(base_directory, posterior, 'dynesty_results'), analysis.varied_parameters, results)
     eos.data.ImportanceSamples.create(os.path.join(base_directory, posterior, 'samples'), analysis.varied_parameters, samples, weights)
 
 
 # Corner plot
 @task('corner-plot', '{posterior}/plots')
-def corner_plot(analysis_file:str, posterior:str, base_directory:str='./'):
+def corner_plot(analysis_file:str, posterior:str, base_directory:str='./', file_extension:str='.pdf'):
```

Comment:
This should probably be a fourth commit then, since it is not fixing any issues in the ``corner_plot`` task but adds new functionality.